### PR TITLE
Add PSC04 test

### PR DESCRIPTION
**Pull request by tlawrencech**
-Adds PSC04 test
-First appoints a PSC01 as part of the test
-Tried a different approach of selecting the change button to expand each section and then cancel button, as there is no need to actually change the data but just load each section

### CODE REVIEW

File: cypress/integration/change_psc/change_individual_psc_details.spec.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+describe('Change of person with significant control (PSC) details', () => {
+    it('File successful PSC04', () => {
+
+        const companyOverview = new CompanyOverviewPage();
+        const preFilingPage = new PreFilingPSCPage();
+        const appointPSC01Page = new AppointPSC01Page();
+        const allForms = new AllFormsPage();
+        const psc04Page = new ChangeIndividualPscPage();
+
+        // Go to PSC01
```
Comment (rsuller) [inline cypress/integration/change_psc/change_individual_psc_details.spec.js]: OK so it's not ideal to have to run this before running the PS04 test but if it's required then maybe we structure the file differently to show that this is required prior to be able to test the PSC04 e.g. add the code to a `before()` method.
See [here](https://docs.cypress.io/guides/core-concepts/writing-and-organizing-tests.html#Hooks)

Comment (tlawrencech) [reply cypress/integration/change_psc/change_individual_psc_details.spec.js]: Restructured adding the PSC01 appointment to a beforeEach method.

Comment (rsuller) [reply cypress/integration/change_psc/change_individual_psc_details.spec.js]: OK, This actually causes us issues when running the entire test suite.
See the order of hooks here https://docs.cypress.io/guides/core-concepts/writing-and-organizing-tests.html#Hooks

Scrap what I said and revert to what you had. Sorry 🙄

Comment (tlawrencech) [reply cypress/integration/change_psc/change_individual_psc_details.spec.js]: Ok just reverted those changes and pushed

**Pull request merged**